app/sections/about/about.component.ts

```diff
@@ -18,53 +18,33 @@ import {transformResponse} from '../../shared/tools/response.tools';
   pipes: [MarkdownPipe]
 })
 export class About implements OnActivate {
-  private content: any;
-  private submenuItems: any[] = [];
+  private content: NodePageContent;
+  private submenuItems: NodePageContent[] = [];
 
   constructor(private _routerData: RouteData,
               private _params: RouteParams,
-              private _contentful: ContentfulService) {
+              private _contentfulContent: ContenfulContent) {
   }
 
   routerOnActivate(to: ComponentInstruction, from: ComponentInstruction): any {
```

Comment:
Let's stick to OnInit - it is available for all components (unlike routerOnActivate - this one hook is for router components). What do you think?

Do we really need Promise in here? I've tried to not use it for couple components in conjunction with OnInit and all was fine.